.coderabbit.yaml

```
@@ -1,67 +1,218 @@
-      - Suggest splitting or refactoring services/controllers that are too large.
-      - Encourage use of DTOs, validation pipes and proper error handling.
-      - Check for security issues: validation, guards, unsafe queries, etc.
-      - Use dependency injection properly and avoid tight coupling.
-      - Suggest extracting shared logic into reusable providers/helpers where appropriate.
-
-tools:
-  eslint:
-    enabled: true
-  gitleaks:
-    enabled: true
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)